### ISSUE TITLE
Control Optimus output size 

### ISSUE THREAD

**Issue by PedroEFLourenco** [pjebs/optimus-go#9]
Is it possible to limit the digit number of the optimus obfuscation output? 
Like, generate obfuscated ids with up to 7digits. 

I understand this would limit the number of possible ids, but it would still be helpful for my use-case.

**Comment by pjebs** [comment]
Try and modify `MAX_INT` to `2^N`.

**Comment by PedroEFLourenco** [comment]
I don't really know what that means in practical terms, my math is pretty rusty. 
That being said, I changed it to the upper bound of numbers I wanted (9999999), and ran the example like this (ignore the package names, its just because I copied the code to some other project I have for testing stuff):

```
mod_inverse := optimus_go.ModInverse(4161499)
o := optimus_go.New(4161499, mod_inverse, 493244)

new_id := o.Encode(15) 
fmt.Println(new_id)

orig_id := o.Decode(new_id)
fmt.Println(orig_id)
```
	
Which didn't really work, since it gave this output:
```
10457833
8951311
```

So either I am doing something wrong, or it didn't work as expected.

**Comment by pjebs** [comment]
It has to be a power of 2.

**Comment by PedroEFLourenco** [comment]
Thank you for the reply. 
Tried it now with the same values but with MAX_INT set to the highest power of 2 that has 7 or less (8881852), but it still didn't work as expected

The results were:

```
8881852
0
```

**Comment by pjebs** [comment]
Can you show me on go playground

**Comment by PedroEFLourenco** [comment]
Yes, sure. Here it is: https://go.dev/play/p/op2AXGK8_xm

**Comment by pjebs** [comment]
I mean your code

**Comment by PedroEFLourenco** [comment]
Yes, the code is in the main function inside the go playground I linked above.

**Comment by pjebs** [comment]
Maybe the mod function assumes int64

**Comment by hzmsrv** [comment]
how to support int64?
